test-built

```diff
@@ -1,11 +1,6 @@
 #!/bin/bash
 
-cd dist
-if [ ! -d bundle ]
```

Comment:
Why remove this?  The bundle won't be unzipped unless one has run the deployment right?  That's why I have this script automatically check if the bundle has been unzipped